kod/object/active/holder/nomoveon/battler/player/user/escapedconvict.kod

```diff
@@ -171,6 +179,115 @@ messages:
       propagate;
    }
 
+   GetName()
+   {
+      % Give a generic resource Clients will not have the proper name resource.
+      % as EscapedConvict is hidden from the who list
+      return escaped_convict_name_rsc;
+   }
+
+   AdminReset()
+   {
+      
+      piMight = 50;
+      piIntellect = 50;
+      piStamina = 50;
+      piAgility = 50;
+      piMysticism = 50;
+      piAim = 50;
+
+      %piMightMod = 20;
+      %piIntellectMod = 20;
+      %piStaminaMod = 20;
+      %piAgilityMod = 20;
+      %piMysticismMod = 20;
+      %piAimMod = 20;
+
+      Send(self,@FillHealth);
+      Send(self,@FillMana);
+      Send(self,@FillVigor);
+
+      Send(self,@GivePlayerAllSpells,#override=TRUE);
+      Send(self,@GivePlayerAllSKills,#override=TRUE);
+      Send(self,@EvaluatePKStatus);
+      Send(self,@PlayerIsIntriguing);
+
+      Send(self,@EquipEscapedConvict);
+
+      return;
+   }
+
+   EquipEscapedConvict()
+   {
+      if (Send(self,@FindHolding,#class=&ReagentBag) = $)
+      {
+         Send(self,@Newhold,#what=Create(&ReagentBag));
+      }
+
+      return;
+   }
+
+   AdminHarm(percent=10)
+   {
+      piHealth = piHealth - ((piHealth / 10) * 100);
```

Comment:
This should be piHealth = piHealth - (piHealth / 10); since it's already in precision units.